index.html

```diff
@@ -59,8 +59,23 @@
     </a>
 
     <!-- Hero -->
-    <section id="hero" class="section-pad-left">
-
+    <section id="hero">
+        <div id="hero-content">
+            <h1>BrickHack 8</h1>
+            <h3>March 5-6, 2022</h3>
+            <h3>RIT Online Hackathon</h3>
```

Comment:
nit: Should these be `<h2>`s to maintain hierarchy? Not a major issue either way

Reply:
Short answer yes but no

Semantically the hierarchy should be h1 -> h2 -> h3

But for simplicity with styling, I matched each semantic heading to a different style guide style. 

If you really want to change it, we need to refactor those styles from being on the elements to being classes in the general pr

Reply:
:+1: that makes sense
I guess semantically they shouldn't be headers at all, but I'll let you go with whatever's best/easiest